plugins/runekit

```diff
@@ -1,3 +1,3 @@
 repository=https://github.com/ImToxicCode/runekit.git
-commit=3ae3e07e2bbe0381166c3508c96bd09373f364ac
+commit=7955515ae39b8db43ade5dae2a9ff2c7280a9a75
 warning=This plugin communicates the stage of crops and birdhouses to a 3rd party server not controlled or verified by the RuneLite Developers.
```

Comment:
This should be updated. Perhaps you should make it more general so it doesn't need to be changed each time a feature is added?

Reply:
I updated the file.